store.go

```diff
@@ -1508,9 +1508,8 @@ func (s *store) Version() ([][2]string, error) {
 }
 
 func (s *store) Mount(id, mountLabel string) (string, error) {
-	rcstore, err := s.ContainerStore()
-	if err != nil {
-		return "", err
+	if layerID, err := s.ContainerLayerID(id); err == nil {
+		id = layerID
 	}
```

Comment:
Shouldn't an error be returned here if `ContainerLayerID` returns an error, instead of falling through and letting the layer store's `Mount()` method eventually return an `ErrLayerUnknown` error?